evennia/contrib/rpg/traits/traits.py

```diff
@@ -171,25 +171,26 @@ class Object(DefaultObject):
 ```
 ## Static trait
 
-`value = base + mod`
+`value = (base + mod) * mult`
 
-The static trait has a `base` value and an optional `mod`-ifier. A typical use
+The static trait has a `base` value and an optional `mod`-ifier and 'mult'-iplier. A typical use
```

Comment:
Worth pointing out that `mult` defaults to `1.0` and can thus be ignored if you don't need it.